internal/pkg/cli/storage_init.go

```diff
@@ -330,14 +328,6 @@ func (o *initStorageOpts) validateStorageType() error {
 		ws:           o.ws,
 		workloadName: o.workloadName,
 	}); err != nil {
```

Comment:
Do we want to mention "vpc connector" at all in the error message? Right now if the usre try to use RDS addon for an app runner service - and of course they wouldn't have `network` in the manifest - Copilot will say "you need a VPC connector" which doesn't make sense to them because, what is a VPC connector anyway 🤔

Reply:
It says `requires a VPC connection` which I think should be probably fine? As it is indeed the reason why they cannot use RDS addon 🤔